modules/manage/pages/use-admin-api.adoc

```diff
@@ -0,0 +1,128 @@
+= Manage Redpanda using the Admin API
+:page-categories: Management, High Availability, Upgrades
+:description: Manage components of a Redpanda cluster, such as individual brokers and partition leadership. The Redpanda Admin API also allows you to perform operations that are specific to Redpanda Self-Managed and cannot be done using the standard Kafka API.
+
+The Redpanda Admin API allows you to manage your cluster and perform operations specific to Redpanda Self-Managed that are not available through the standard Kafka API. You can call the Admin API using any HTTP client. 
+
+Most Admin API operations are also available using xref:get-started:intro-to-rpk.adoc[`rpk`], a CLI tool which interacts with the Admin API under the hood. 
+
+Redpanda version 25.3 introduces a version 2 of the Admin API, powered by https://connectrpc.com/docs/introduction[ConnectRPC]. New Redpanda features and operations introduced starting in version 25.3 are available with Admin API v2 only, and are not backported to Admin API v1. Supported Admin API v1 operations remain available and you can continue to use them per usual (including with rpk v25.3 and later).
+
+== Prerequisites
+
+* A running Redpanda Self-Managed cluster.
+** For Admin API v1, any supported version of Redpanda. For Admin API v2, Redpanda version 25.3 or later.
+* Superuser privileges, if xref:manage:security/authentication.adoc#enable-authentication[authentication] is enabled on your cluster for the Admin API. For more information, see xref:manage:security/authentication.adoc#create-superusers[Configure Authentication]. (Some endpoints are read-only and do not require superuser access.)
+* A tool to make HTTP requests, such as `curl`, or client libraries for your programming language of choice. 
+** For Admin API v2, you can also make requests using a Connect client. You can install the Connect plugin for your preferred language and use the Protobuf compiler to generate an SDK. Admin API v2 is also available as a Buf module and can be accessed through the https://buf.build/redpandadata/core/docs/dev:redpanda.core.admin.v2[Buf Schema Registry]. The https://buf.build/docs/cli/[Buf CLI] provides an easy way to generate client SDKs.
```

Comment:
Is this a Kafka Connect? If you are referring generally to connectors, wouldn't this be "connect"?
Or is this Redpanda Connect?